nsv13/code/game/gamemodes/overmap/objectives/rubicon.dm

```diff
@@ -38,8 +39,9 @@
 /datum/overmap_objective/clear_system/rubicon
 	system_name = "Rubicon"
 	extension_supported = TRUE
-	required_players = 4
+	required_players = 10
 
 /datum/overmap_objective/clear_system/dolos
 	system_name = "Dolos Remnants"
-	required_players = 4
+	extension_supported = TRUE //Only if Rubicon is not available
```

Comment:
This var doesn't actually do anything as the only section using it is commented out.

Reply:
Yeah I've just set it because the variable exists and I want it to be included in a future system that uses it.